apps/voting/contracts/Voting.sol

```diff
@@ -264,7 +260,7 @@ contract Voting is IForwarder, AragonApp {
         ExecuteVote(_voteId);
     }
 
-    function _isVoteOpen(Vote storage vote) internal view returns (bool) {
+    function _isVoteOpen(Vote vote) internal view returns (bool) {
```

Comment:
I'm pretty sure the `storage` is a small optimization to avoid copying the entire struct into memory when we call this function. I would hope the compiler errors if we don't pass in a storage variable to this function, but who knows with solc.